base/linalg/transpose.jl

```diff
@@ -126,7 +126,6 @@ function ctranspose(A::AbstractMatrix)
     ctranspose!(B, A)
 end
 
-@inline ctranspose(A::AbstractVector{<:Real}) = transpose(A)
```

Comment:
Yes, your understanding is the same as mine. I think you're right about the definitions though—I think the `RowVector` one should be the one that's removed, since this method that calls `transpose` should end up [becoming a `RowVector` anyway](https://github.com/JuliaLang/julia/blob/9706c8b67fe03dd859cbb01427ab0e618414ca46/base/linalg/rowvector.jl#L76). I'll make that change. Thanks for your help!